src/ui/components/Rating/index.js

```diff
@@ -93,11 +94,9 @@ export class RatingBase extends React.Component {
         `possible values: ${RATING_STYLE_SIZES.join(', ')}`);
     }
     let description;
-    if (rating) {
+    if (rating && readOnly) {
```

Comment:
Thanks for this! A comment here explaining why only `readOnly` rating wrappers should get this description might be good here.